.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
 
 ## Process & policy checklist
 <!--- Review the list and check the boxes that apply. -->
-- [ ] I have added tests for my changes.
-- [ ] I have tested my changes.
-- [ ] I have updated the project documentation to reflect my changes.
-- [ ] I have read the CONTRIBUTING documentation and followed the standards set for this project.
 
- 
-Click here for details on [formatting pull requests](https://microsoft.github.io/fast-dna/docs/contributing/working#formatting-messages).
+- [] I have added tests for my changes.
+- [] I have tested my changes.
+- [] I have updated the project documentation to reflect my changes.
+- [] I have read the CONTRIBUTING documentation and followed the standards set for this project.
+
+## Pull request formatting
+
+**`<type>`** is required to be at least one of the following:
```

Comment:
@awentzel this reads odd to me in this context. Conventional commits will only pickup one of these, so "at least" seems out of place. How would I enumerate both a fix and feature?